docs/getting-started/db.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Preparing the Database
+---
+
+# Preparing the Database
+
+We need to create our database, load the database schema, and load seed
```

Comment:
FYI: the installation guides all end with the instruction to run `bin/setup` which also initializes the DB, so at this point the user might just have already everything set up